WcaOnRails/app/models/round.rb

```diff
@@ -4,6 +4,10 @@ class Round < ApplicationRecord
   has_one :event, through: :competition_event
   belongs_to :format
 
+  serialize :time_limit, TimeLimit
+  serialize :combined_round_cutoff, CombinedRoundCutoff
+  serialize :next_round_cutoff, Cutoff
```

Comment:
@larspetrus, that's frustrating. Let me know if you'd like me to try to explain the relevant parts of the regulations to you sometime. Totally fine if you don't want to bother with this, though.

I had a chat with @Laura-O about this, and we came up with what I think is a clever solution to this! I think the right way to describe this is that rounds can have a "qualification", which is some requirement you have to satisfy in order to be eligible to compete in that round. Normally this means you're in the top N people from the previous round. This raises the question "what does it mean to have a qualification on round 1"? Well, that's a requirement you have to satisfy in order to be able to register for that event at that competition (see [regulation Z4](https://www.worldcubeassociation.org/regulations/#Z4) and [US Nationals for this summer](https://www.cubingusa.com/nationals2017/qualifying.php))